pkg/api/v1alpha1/clustercurator_types.go

```diff
@@ -0,0 +1,85 @@
+// +kubebuilder:object:generate=true
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	runtime "k8s.io/apimachinery/pkg/runtime"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// ClusterCuratorSpec defines the desired state of ClusterCurator
+type ClusterCuratorSpec struct {
+	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
+	// Important: Run "make" to regenerate code after modifying this file
+
+	// This is the desired curation flow
```

Comment:
Is this value can be any value, or it should be fixed to several options? need more explanation in the comment

Reply:
The text was update to include the curation flow, as well the field has been enumerated to `install, scale, upgrade, & destroy`